tests/test_providers.py

```diff
@@ -44,6 +44,8 @@ def token(mocker):
 def mock_network(mocker):
     mock = mocker.MagicMock(spec=NetworkAPI)
     mock.name = "MOCK_NETWORK"
+    mock.ecosystem = mocker.MagicMock(spec=EcosystemAPI)
+    mock.ecosystem.name = "MOCK_ECOSYSTEM"
```

Comment:
Seems like this should replicate a real ecosystem name?

Reply:
I dont think it really matters but you could

Reply:
oh yeah that is why the tests are failing, so yeah make this real